installing/installing_ibm_cloud_power_vs/configuring-iam-ibm-cloud-power-vs.adoc

```diff
@@ -0,0 +1,29 @@
+:_content-type: ASSEMBLY
+[id="configuring-iam-ibm-cloud"]
+= Configuring IAM for IBM Cloud
+include::_attributes/common-attributes.adoc[]
+:context: configuring-iam-ibm-cloud
+
+toc::[]
+
+In environments where the cloud identity and access management (IAM) APIs are not reachable, you must put the Cloud Credential Operator (CCO) into manual mode before you install the cluster.
+
+include::modules/alternatives-to-storing-admin-secrets-in-kube-system.adoc[leveloffset=+1]
+
+[role="_additional-resources"]
+[id="additional-resources_configuring-iam-ibm-cloud-about-cco"]
+.Additional resources
+* xref:../../authentication/managing_cloud_provider_credentials/about-cloud-credential-operator.adoc#about-cloud-credential-operator[About the Cloud Credential Operator]
+
+include::modules/cco-ccoctl-configuring-power-vs.adoc[leveloffset=+1]
+//include::modules/manually-maintained-credentials-upgrade.adoc[leveloffset=+1]
+// Will need to revisit upgrade scenario for IBM Cloud; not needed until OCP 4.11. Tentative instructions have been added for reference later.
+
+[role="_additional-resources"]
+[id="additional-resources_configuring-iam-ibm-cloud-refreshing-ids"]
+.Additional resources
+* xref:../../post_installation_configuration/cluster-tasks.adoc#refreshing-service-ids-ibm-cloud_post-install-cluster-tasks[Rotating API keys for IBM Cloud VPC]
```

Comment:
We should have a section specific to Power VS in `post_installation_configuration/cluster-tasks` rather than point to the IBM Cloud section. The command for Power VS is:
```
$ ccoctl powervs refresh-keys \
    --kubeconfig <openshift_kubeconfig_file> \ 
    --credentials-requests-dir <path_to_credential_requests_directory> \ 
    --name <name>
```